tree/ntuple/v7/src/RField.cxx

```diff
@@ -557,8 +588,9 @@ void ROOT::Experimental::RField<std::uint16_t>::GenerateColumnsImpl()
       std::uint16_t, EColumnType::kInt16>(model, 0)));
 }
 
-void ROOT::Experimental::RField<std::uint16_t>::GenerateColumnsImpl(const RNTupleDescriptor &)
+void ROOT::Experimental::RField<std::uint16_t>::GenerateColumnsImpl(const RNTupleDescriptor &desc)
 {
+   EnsureColumnType({EColumnType::kInt16}, 0, desc);
```

Comment:
We _might_ care about signedness here but it is for another PR.  The column type is the same (because it uses the same twos complement bit representation) but the full column model on disk does store in a flag whether or not the column contains negative values.  And we probably should check that information somewhere.